concepts/05 UI Components/Diagram/10 Data Binding/40 Optional Binding Expressions.md

```diff
@@ -162,6 +162,8 @@ The Diagram allows you to bind a number of shape and connector visual properties
 
 You can set a binding property to a name of a data source field that provides item values, or to an expression that returns a constant value or calculates a value in runtime based on conditions.
```

Comment:
We should rewrite this sentence without using "provides". Can we say "supplies"?

Also, please change "a name of a data source field" to "the name of a data source field".

Reply:
updated